mcs_benchmark_data/pipelines/commonsense_qa/kagnet_commonsense_qa_submission_pipeline.py

```diff
@@ -13,20 +13,22 @@
 
 
 class KagnetCommonsenseQaSubmissionPipeline(_Pipeline):
-    ID = "CommonsenseQA"
-    SUBMISSION_NAME = "kagnet"
+    BENCHMARK_ID = "CommonsenseQA"
+    SUBMISSION_ID = "kagnet"
 
     def __init__(self, file_names: KagnetCommonsenseQaSubmissionFileNames, **kwds):
         _Pipeline.__init__(
             self,
             extractor=BenchmarkSubmissionExtractor(
-                pipeline_id=self.ID,
+                pipeline_id=self.BENCHMARK_ID,

```

Comment:
Sorry, I meant that pipeline_id would also be benchmark_id. But fine to leave this for now.